Ci fails when AKS is not deployed

<!-- IF SUFFICIENT INFORMATION IS NOT PROVIDED VIA THE FOLLOWING TEMPLATE THE ISSUE MIGHT BE CLOSED WITHOUT FURTHER CONSIDERATION OR INVESTIGATION -->

## System Information

- OS and Version: 
- Software Version:

## Expected Behavior

<!-- Tell us what should happen -->

## Current Behavior

<!-- Tell us what happens instead of the expected behavior -->
https://github.com/ks6088ts-labs/baseline-environment-on-azure-terraform/actions/runs/12969923140/job/36196466642

## Steps to Reproduce

<!-- Provide a link to a live example, or an specific set of steps to -->

1. Run CI test

## Logs

<!-- Provide logs if possible -->

```
╷
│ Error: Invalid index
│ 
│   on outputs.tf line 7, in output "aks_cluster_name":
│    7:   value       = module.aks[0].name
│     ├────────────────
│     │ module.aks is empty tuple
│ 
│ The given key does not identify an element in this collection value: the
│ collection has no elements.
╵
Error: Terraform exited with code 1.
make[1]: *** [Makefile:77: plan] Error 1
make[1]: Leaving directory '/home/runner/work/baseline-environment-on-azure-terraform/baseline-environment-on-azure-terraform/infra'
make: *** [Makefile:88: ci-test] Error 1
Error: Process completed with exit code 2.
```

## Additional Information

<!-- Any additional information that might be useful -->

--- a/infra/scenarios/workshop_azure_openai/outputs.tf
+++ b/infra/scenarios/workshop_azure_openai/outputs.tf
@@ -4,6 +4,6 @@ output "resource_group_name" {
 }
 
 output "aks_cluster_name" {
-  value       = module.aks[0].name
+  value       = length(module.aks) > 0 ? module.aks[0].name : ""
   description = "created Azure Kubernetes Service name"
 }

